### PR TITLE
Add COMEBin

### PR DESCRIPTION
**Pull request by ziyewang**
add COMEBin recipes

add a conda recipe for COMEBin, a metagenomics contig binner (https://github.com/ziyewang/COMEBin).

### CODE REVIEW

File: recipes/comebin/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+source:
+  url: https://github.com/ziyewang/COMEBin/archive/refs/tags/1.0.2.tar.gz
+  sha256: 8e1bfe609f723743f7e1703dfd89483a8a4def19b9e9a3283839c7eb7c93d1ab
+
+build:
+  number: 0
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage('comebin', max_pin="1") }}
```
Comment (thanhleviet) [inline recipes/comebin/meta.yaml:13]: `- {{ pin_subpackage('comebin', max_pin="x.x") }}`

**Comment by ziyewang** [comment]
@BiocondaBot please fetch artifacts

**Comment by ziyewang** [comment]
@BiocondaBot please add label

**Comment by ziyewang** [comment]
@BiocondaBot please add label

**Comment by mencian** [comment]
Thanks, @ziyewang!

**Review by mencian — APPROVED** [review]

**Pull request merged**